offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/OfferToPDFConverter.groovy

```diff
@@ -245,22 +327,49 @@ class OfferToPDFConverter implements OfferExporter {
     private static class ItemPrintout {
 
         static String itemInHTML(int offerPosition, ProductItem item) {
+            String totalCost = Currency.getFormatterWithoutSymbol().format(item.quantity * item.product.unitPrice)
             return """<div class="row product-item">
                         <div class="col-1">${offerPosition}</div>
                         <div class="col-4 ">${item.product.productName}</div>
                         <div class="col-1 price-value">${item.quantity}</div>
                         <div class="col-2 text-center">${item.product.unit}</div>
                         <div class="col-2 price-value">${Currency.getFormatterWithoutSymbol().format(item.product.unitPrice)}</div>
-                        <div class="col-2 price-value">${Currency.getFormatterWithoutSymbol().format(item.quantity * item.product.unitPrice)}</div>
+                        <div class="col-2 price-value">${totalCost}</div>
                     </div>
                     <div class="row product-item">
                         <div class="col-1"></div>
                         <div class="col-4 item-description">${item.product.description}</div>
                         <div class="col-7"></div>
-                    </div>"""
+                    </div>
+                    """
+
         }
 
-        static String tableHeader(String elementId){
+        static String subTotalFooter(boolean isOverhead) {
+            String overheadStatus = ""
+            if (isOverhead) {
+                overheadStatus = "overhead"
+            }else {
+                overheadStatus = "no-overhead"
+            }
```

Comment:
instead of this you could declare two class variables that you can provide for the method. This would make sure that the strings don't change and you can avoid the boolean parameter for the method

Reply:
Good Idea!